tractor/_actor.py

```diff
@@ -832,9 +832,10 @@ async def _serve_forever(
                         host=accept_host,
                     )
                 )
-                log.debug("Started tcp server(s) on"  # type: ignore
-                          f" {[l.socket for l in listeners]}")
-                self._listeners.extend(listeners)
+                log.debug(
+                    "Started tcp server(s) on"
+                    f" {[getattr(l, 'socket', 'unknown socket') for l in l]}")
```

Comment:
just `mypy` stuff since technically `.socket` isn't discovered through the type system easily.